website_wow/static/src/js/website_wow.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2017 LasLabs Inc.
+   License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+*/
+
+odoo.define('website_wow.website_wow', function (require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    var core = require('web.core');
+
+    var WebsiteWow = {
```

Comment:
Well the issue is that they allow a paradigm of templates changing the selectors that are used. From what I understand of these template variables, they are not accessible in my Javascript, which is what would need to happen for a template to be able to update the targeted selector by using only XML